dev/reference/tools/data-migration/from-aurora.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Data Migration from AWS Aurora MySQL to TiDB
+summary: Using DM to migrate from AWS Aurora MySQL to TiDB
+category: tools
+---
+
+# Data Migration from AWS Aurora MySQL to TiDB
+
+This document describes how to use DM to migrate from [AWS Aurora MySQL](https://aws.amazon.com/cn/rds/aurora/details/mysql-details/) to TiDB。
+
+## Step 1: Start binlog in Aurora cluster
+
+This document assumes you'll migrate data from two Aurora clusters to  TiDB. The information list of Aurora clusters is as belowed. It indicates Aurora-1 has a single built-in reader endpoint. 
+
+| Cluster | Endpoint | Port | Role |
+|:-------- |:--- | :--- | :--- |
+| Aurora-1 | pingcap-1.h8emfqdptyc4.us-east-2.rds.amazonaws.com | 3306 | Writer |
+| Aurora-1 | pingcap-1-us-east-2a.h8emfqdptyc4.us-east-2.rds.amazonaws.com | 3306 | Reader |
+| Aurora-2 | pingcap-2.h8emfqdptyc4.us-east-2.rds.amazonaws.com | 3306 | Writer |
+
+During the incremental backup process, you need to set the binglog format as `ROW`. Inactive binlog or incorrect binlog format will hinder data migration using DM. See [Checking items](https://www.pingcap.com/docs-cn/tools/dm/precheck/#%E6%A3%80%E6%9F%A5%E5%86%85%E5%AE%B9) for more information.
+
+> **Warning:** 
+>
+> Aurora reader cannot start binlog, so it cannot be the upstream master server when using DM to migrate data.
+
+If you need to use GTID for replication, enable GTID for Aurora cluster. 
+
+> **Warning:**
+>
+> GTID-based data migration requires MySQL 5.7 (Aurora 2.04.1) version or later. 
+
+### Modify binlog relevent parameters in the Aurora cluster 
+
+In Aurora cluster, binlog relevant parameters are in the **cluster parameter group**. See [Enable Binary Logging on the Replication Master](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Replication.MySQL.html) for more information about enabling binlog. You need to set the `binlog_format` in `ROW` when using DM for migration. 
```

Comment:
- For a countable noun, either add an article before it or use the plural form.
- Some meaning is lost.
- The link is not accurate enough.
- set the `binlog_format` in `ROW`: `to` or `as`, but not `in`

```suggestion
In the Aurora cluster, binlog related parameters are cluster level parameters among cluster parameter groups. For more information about enabling binlog in the Aurora cluster, see [Enable Binary Logging on the Replication Master](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Replication.MySQL.html#AuroraMySQL.Replication.MySQL.EnableBinlog). You need to set the `binlog_format` to `ROW` when using DM for data migration.
```